core/src/main/resources/org/apache/commons/jelly/tags/Resources.properties

```diff
@@ -228,10 +228,10 @@ SQL_DATASOURCE_NULL=\
     'dataSource' is null
 
 SQL_MAXROWS_PARSE_ERROR=\
-    Error parsing 'javax.servlet.jsp.jstl.sql.maxRows' configuration setting: "{0}"
+    Error parsing 'jakarta.servlet.jsp.jstl.sql.maxRows' configuration setting: "{0}"
 
 SQL_MAXROWS_INVALID=\
-    'javax.servlet.jsp.jstl.sql.maxRows' configuration setting neither an Integer nor a String
+    'jakarta.servlet.jsp.jstl.sql.maxRows' configuration setting neither an Integer nor a String
```

Comment:
We don't use `jelly-sql` anywhere I think?

Reply:
Correct. This change is only for logical consistency with the `jakarta.servlet.jsp.jstl-api` bump.